packages/scripts/turbo/generators/config.ts

```diff
@@ -41,14 +41,14 @@ export default function generator(plop: PlopTypes.NodePlopAPI): void {
 			{
 				type: 'addMany',
 				destination: `${plop.getDestBasePath()}/../{{name}}`,
-				templateFiles: ['templates/**'],
+				templateFiles: ['templates/**', '!templates/default/cliff.toml'],
```

Comment:
Can you add a comment here to make sure no one removes this in the future accidentally. I presume it's because the toml replacer syntax cliff uses causes problems with the default plop template replacer....